src/autocomplete.js

```diff
@@ -62,6 +62,7 @@ class Autocomplete {
         this.keyboardHandler.bindKeys(this.commands);
         this.parentNode = null;
         this.setSelectOnHover = false;
+        this.stickySelectionDelay = 500;
```

Comment:
Can we add JSDoc comment somehow. for this option, since it's a bit hard to understand from just the name without prior context, but it's a pretty cool functionality.

Reply:
added a (hopefully) descriptive comment about this feature